sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -4,6 +4,8 @@
 
 ```ts
 
+/// <reference types="node" />
```

Comment:
This is a bit concerning, going back to our discussion about how our packages just won't work without types/node. Does this change also manifest in the .d.ts? I had upgraded a few versions for the next FR releaase as well only in that working branch, and saw this there too.

Reply:
Hmmm great question Will! It _does_ manifest in the .d.ts rollup; however, I should note that it shows up there _today_ without these changes 
![image](https://user-images.githubusercontent.com/753570/135346381-3eae27a6-9993-4881-a415-44bbf6e466e3.png)